src/main/resources/explanations/challenge46_reason.adoc

```diff
@@ -0,0 +1,7 @@
+*Why Vault Template Injection is not always a good idea?*
+
+While Vault agent injection via templates can be a convenient way to manage secrets in certain scenarios,
+there are situations where it might not be the best approach.
+
+Templates might accidentally expose sensitive information in logs or temporary files.
+If not properly configured, secrets could end up in places where they are accessible by unauthorized users or processes.
```

Comment:
maybe we can add an example here where the data of the template (in a scripted langauge) can lead to an injection attack of the language/app ot which it gets injected itself? (E.g. a php string concatenation issue)

Reply:
done